hoa2dot/core.py

```diff
@@ -459,7 +490,10 @@ def __eq__(self, other):
 
     def __hash__(self):
         """Define hash for State."""
-        return hash((self.index, self.label, self.name, self.name, self.acc_sig))
+        try:
+            return hash((self.index, self.label, self.name, self.name, self.acc_sig))
+        except TypeError:
+            pass
```

Comment:
why did you add it? is there a case in which the hash could not be computed?

Reply:
ops, my bad, that was added for debugging...